modules/reminders/send.ts

```diff
@@ -194,6 +195,37 @@ async function sendReminders(): Promise<NodeJS.Timeout | undefined> {
 					await syncRandomBoard();
 					continue;
 				}
+				case SpecialReminders.ChangeStatus: {
+					let status = [
+						"Watching the SA server!",
+						"Watching for bugs...",
+						"Dating Callum",
+						"Yes",
+						"Auto-Moderating the SA Server!",
+						"Farming dangos",
+						"Rico, status",
+						"Scanning potatoes",
+					];
+					const next = (Number(reminder.reminder) + 1) % status.length;
+
+					remindersDatabase.data = [
+						...remindersDatabase.data,
+						{
+							channel: "0",
+							date: Number(Date.now() + 18_000_000),
```

Comment:
```suggestion
							date: Date.now() + ((Math.random() * 3) + 3) * 3_600_000,
```
to randomly change every 3-6 hours (i changed my mind again)
(i like randomness)

Reply:
lol